server/controllers/wazuh-elastic.js

```diff
@@ -41,8 +41,7 @@ export class WazuhElasticCtrl {
       'auditing-policy-monitoring': [{ rootcheck: true }, { audit: true }, { openscap: true }, { ciscat: true }],
       'threat-detection': [{ vulnerabilities: true }, { virustotal: true }, { osquery: true }, { docker: true }, { mitre: true }]
     };
-    this.wzSampleAlertsIndexPrefix = WAZUH_SAMPLE_ALERT_PREFIX;
-    this.buildSampleIndexByCategory = (category) => `${this.wzSampleAlertsIndexPrefix}sample-${category}` // wazuh-alerts-sample-security, wazuh-alerts-sample-auditing-policy-monitoring, wazuh-alerts-threat-detection
+    this.buildSampleIndexByCategory = (indexPattern, category) => `${indexPattern.replace(/\*/g, '' )}-sample-${category}` // ie for wazuh-alerts-* -> wazuh-alerts.x-sample-security
```

Comment:
Review the comment:
```js
// ie for wazuh-alerts-* -> wazuh-alerts.x-sample-security
```

It should be, because is adding `-sample-${category}`:
```js
// ie for wazuh-alerts-* -> wazuh-alerts--sample-security
```